README.md

```diff
@@ -166,6 +166,51 @@ node apply-interest.js
 
 It is recommended to run this script once per month.
 
+### Tracking Home Prices (RentCast's value estimate)
+
+This script tracks the RentCast value for a home.  It adds new transactions
+to keep the account balance equal to the latest value.  Rentcast values differ
+from Zillow since they don't have as complete a database, but they are close in
+most cases.
+
+To use this script, you need to create a new account in Actual Budget and set
+the account note to populate the fields that RentCast needs: address, bedrooms, 
+bathrooms, squareFootage, and optionally propertyType and/or compCount.  Values with 
+spaces and special characters need to be URL encoded, an online encoder 
+like https://www.urlencoder.org/ is helpful.
+
+"address" needs be one line with commas seperating the address lines, then URL encoded.  
+For example, "123 Example, St Anytown, CA ,12345"
+```
+address:123%20Example%2C%20St%20Anytown%2C%20CA%20%2C12345
+bedrooms:4
+bathrooms:2
+squareFootage:1600
```

Comment:
```suggestion
```

Reply:
these are basically required, I'll expand the text above it